Content.Server/Physics/Controllers/ConveyorController.cs

```diff
@@ -96,7 +97,7 @@ private void UpdateAppearance(ConveyorComponent component)
         {
             if (!EntityManager.TryGetComponent<AppearanceComponent?>(component.Owner, out var appearance)) return;
             var isPowered = this.IsPowered(component.Owner, EntityManager);
```

Comment:
```suggestion
        {
            var isPowered = this.IsPowered(component.Owner, EntityManager);
```

You can remove this component check, since you don't use the AppearanceComponent any more and the appearance system itself checks if there is an AppearanceComponent upon SetData.